### PR TITLE
[eslint config] [*] [new] add `eslint` `v7`

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
CC @ljharb

### CODE REVIEW

**Comment by ljharb** [comment]
Thanks, I already have this branch locally tho and it'll require a bit more than this. I'll repurpose the PR.

File: packages/eslint-config-airbnb-base/rules/errors.js [hunk 1]
```diff
@@ -131,13 +136,19 @@ module.exports = {
...
+    'no-useless-backreference': 'off',
+
     // disallow negation of the left operand of an in expression
     // deprecated in favor of no-unsafe-negation
     'no-negated-in-lhs': 'off',
 
     // Disallow assignments that can lead to race conditions due to usage of await or yield
     // https://eslint.org/docs/rules/require-atomic-updates
-    // TODO: enable, semver-major
+    // note: not enabled because it is very buggy
```
Comment (keann) [inline packages/eslint-config-airbnb-base/rules/errors.js:151]: Hi, @ljharb !

Would you mind to share some cases where you faced bugs in this rule?

Comment (ljharb) [reply packages/eslint-config-airbnb-base/rules/errors.js:151]: There’s a couple threads on eslint core that are full of examples; i don’t have any off hand.

Comment (keann) [reply packages/eslint-config-airbnb-base/rules/errors.js:151]: right, might have guess myself to look there. thx

**Pull request merged**